src/api/keystone.test.ts

```diff
@@ -27,14 +28,27 @@ describe('Create an Application', () => {
       },
     });
 
-    // update the application with buyer ID.
+    // create policy and associate with the application.
+    const policy = await policies.create({
```

Comment:
would this be better in a helper to create for us?

Reply:
This is calling a helper function? Same as the other create/update methods in this file:

```
applications.create(...
buyers.create(...
applications.update(...
```